src/C2lcMath.test.js

```
@@ -27,3 +27,10 @@ test('C2lcMath.wrap', () => {
     expect(C2lcMath.wrap(-20, -10, -30)).toBe(-20);
     expect(C2lcMath.wrap(-20, -10, -33)).toBe(-13);
 });
+
+test('Calculating normalized correlation', () => {
+    // javascript rounds-up to 16 decimal degits 
+    expect(C2lcMath.getNormalizedCorrelation([0,2,3],[0,2])).toBe(0);
```

We expect an exception to be thrown here, rather than a return value of 0.